astropy/time/formats.py

```diff
@@ -725,8 +725,9 @@ class TimeUnixTai(TimeUnix):
       ``1969-12-31 23:59:59.999918 UTC``.  The difference between TAI and UTC
       at that epoch was 8.000082 sec.
     - On the day of a leap second the difference between ``unix`` and ``unix_tai``
-      times increases linearly through the day by 1.0.  See also the
-      documentation for the `~astropy.time.TimeUnix` class.
+      times increases linearly through the day by 1.0.
```

Comment:
The original here was also fine, please revert.

Reply:
I reverted the unrelated docs changes and will open a second PR for those.